mithril-aggregator/src/services/snapshotter/compressed_archive_snapshotter.rs

```diff
@@ -106,6 +107,28 @@ impl Snapshotter for CompressedArchiveSnapshotter {
             .await
     }
 
+    async fn compute_immutable_files_total_uncompressed_size(
+        &self,
+        up_to_immutable_file_number: ImmutableFileNumber,
+    ) -> StdResult<u64> {
+        if up_to_immutable_file_number == 0 {
+            return Err(anyhow!(
+                "Could not compute the total and average size without immutable files"
```

Comment:
I think this error message should be updated.
```suggestion
                "Could not compute the total size without immutable files"
```